src/GitVersionCore/ExecuteCore.cs

```diff
@@ -57,7 +58,14 @@ public VersionVariables ExecuteGitVersion(string targetUrl, string dynamicReposi
                 if (versionVariables == null)
                 {
                     versionVariables = ExecuteInternal(targetBranch, commitId, repo, gitPreparer, projectRoot, buildServer, overrideConfig: overrideConfig);
-                    gitVersionCache.WriteVariablesToDiskCache(repo, dotGitDirectory, versionVariables);
+                    try
+                    {
+                        gitVersionCache.WriteVariablesToDiskCache(repo, dotGitDirectory, versionVariables);
+                    }
+                    catch (IOException)
+                    {
+                        Logger.WriteInfo("I/O exception during cache write");
```

Comment:
For consistency IMHO this should write a message with warning level, followd by detailed exception message as info level, as e.g. done [here](https://github.com/GitTools/GitVersion/blob/05161c36debf88b4278b7d57553ccae4c62d1174/src/GitVersionCore/GitVersionCache.cs#L669).